sky/optimizer.py

```diff
@@ -20,6 +20,7 @@
 from sky.utils import env_options
 from sky.utils import log_utils
 from sky.utils import ux_utils
+from sky.utils.resources_utils import DiskTier
```

Comment:
```suggestion
from sky.utils import resources_utils
```
nit: Use `import statements` for packages and modules only, not for individual types, classes, or functions.
https://google.github.io/styleguide/pyguide.html#22-imports